src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -52,7 +52,7 @@ var adyenHelperObj = {
   LOADING_CONTEXT_LIVE:
     'https://checkoutshopper-live.adyen.com/checkoutshopper/',
 
-  CHECKOUT_COMPONENT_VERSION: '3.18.2',
+  CHECKOUT_COMPONENT_VERSION: '3.23.0',
```

Comment:
Was this update also required? If it works with the bundle, then we can revert this change. 
Otherwise, we ideally have to update the bundle as well